docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -75,14 +70,12 @@ Documentation
 * Added gallery example demonstrating how to use
   different Perez coefficients in a ModelChain.
   (:issue:`2127`, :pull:`2148`)
-* Fixed examples in :py:func:`pvlib.shading.shaded_fraction1d`. (:pull:`#2215`)
-
-* Removed unused "times" input from dni_et() function (:issue:`2105`)
+* Fixed examples in :py:func:`pvlib.shading.shaded_fraction1d`. (:pull:`2215`)
 * Updated model description, added equation variable definitions, and created a 
   new "notes" section in :py:func:`pvlib.irradiance.haydavies` and
-  :py:func:`pvlib.irradiance.klucher` (:issue:`2183`, :pull:`2191`,
+  :py:func:`pvlib.irradiance.klucher`. (:issue:`2183`, :pull:`2191`,
   :pull:`2192`)
-* Split the contributing page into several pages (:issue:`2210`, :pull:`2219`)
+* Split the contributing page into several pages. (:issue:`2210`, :pull:`2219`)
 
```

Comment:
```suggestion
* Use Wm⁻² instead of W/m^2 throughout `pvlib.irradiance`. (:issue:`2205`, :pull:`2191`)

```

Reply:
As an user or maintainer, I would prefer a note clarifying "improved units formatting" on the entry already here: https://github.com/kandersolar/pvlib-python/blob/82e47829fb5390887f6ad07778a982f9fa14decb/docs/sphinx/source/whatsnew/v0.11.1.rst?plain=1#L74C1-L77C16